core/src/test/scala/unit/kafka/server/MetadataCacheTest.scala

```diff
@@ -229,7 +205,7 @@ class MetadataCacheTest {
                                                                        errorUnavailableListeners: Boolean): Unit = {
     val topic = "topic"
 
-    val cache = new MetadataCache(1)
+    val cache = new MetadataCache(9)
```

Comment:
nit: maybe it would be clearer if we passed `brokerId` as an argument? Then we could use broker 0 in the test case, for example, so that the UpdateMetadata request makes more sense.

Reply:
@hachikuji done